.github/workflows/staging-smoke-tests.yaml

```diff
@@ -65,3 +75,14 @@ jobs:
           ./gradlew uploadResults --args="BroadJadeDev.json tmp/TestRunnerResults"
 
           cd ${GITHUB_WORKSPACE}/${workingDir}
+      - name: "Notify Jade Slack"
+        if: always()
```

Comment:
I assume this is following current practice, but I'm wondering if `always()` is the best setting here. In lantern and hornet we configured builds to only notify on failure, which makes the ci channel less chatty, and easier to see when problems have occurred. (I think we actually used a setting which notified on all failures and on the first success after a failure, but I'm not sure if github actions has a way to do this.)

Reply:
That's an interesting idea. The recommendation from the QA / devops team was to log both success and failures, so for the GitHub actions I've done that for consistency.

Reply:
Also, this action will likely be fired off from a cron job in the future, and I've been keeping my eyes peeled in the alert channels for these types of actions to pass or fail when they run nightly.

Reply:
I'll bring this up in the next QA / devops meeting.

Reply:
It's true that only notifying on failure means that you don't know if a test never ran. But relying on people to recognize that state isn't good either, since people aren't good at remembering all the tests that they should see, etc.